[idbindex_count4.htm]
  [IDBIndex.count() - throw DataError when using invalid key]
    expected: FAIL

